### ISSUE TITLE
DemoPhysics does not compile

### ISSUE THREAD

**Issue by duerrp** [part-cw/lambdanative#49]
Hi,

I just started playing with your framework and it looks very cool! Thanks for sharing the great work. I tried to compile various examples and it looks like the hash for the chipmunk library is outdated:

```
 => downloading https://chipmunk-physics.net/release/ChipmunkLatest.tgz..
WARNING: wget failed, retrying without certificate check..
 == hash 934a6175560e741e963538356bb7ef2b59323b13
** ERROR: Unable to proceed! package download failed (checksum error)
```
How can I fix this?

**Comment by mgorges** [comment]
Sorry about that! The download link specified a generic latest release and ChipMunk recently was updated. Fixed in 0a37c23, which downloads a specific point release.

**Comment by duerrp** [comment]
Great! Thanks a lot for the quick fix...
